README.md

```diff
@@ -1,8 +1,13 @@
 # Bloxone Azure Templates
 
 ## Getting Started
+ 
+To prepare ARM templates for QA team use patch-for-qa.sh script from utils directory:
+```
+$ utils/archive.sh accessPortal="" configServer="" notificationServer=""
```

Comment:
it should be utils/patch-for-qa.sh right?

Reply:
yes